lib/rex/proto/kerberos/credential_cache/krb5_ccache_presenter.rb

```diff
@@ -0,0 +1,275 @@
+      output << "Key Version Number: #{ticket.enc_part.kvno}".indent(4)
+
+      if key.blank?
+        output << 'Cipher:'.indent(4)
+        output << Base64.strict_encode64(ticket.enc_part.cipher).indent(6)
+      else
+        output << "Decrypted (with key: #{key.bytes.map { |x| "\\x#{x.to_s(16).rjust(2, '0')}" }.join}):".indent(4)
+        output << present_encrypted_ticket_part(ticket, key).indent(6)
+      end
+
+      output.join("\n")
+    end
+
+    # @param [Rex::Proto::Kerberos::Pac::Krb5PacInfoBuffer] info_buffer
+    # @return [String] A human readable representation of a Logon Information
+    def present_logon_info(info_buffer)
+      validation_info = info_buffer.buffer.pac_element.data
+
+      output = []
+      output << 'Validation Info:'
+
+      output << "Logon Time: #{present_time(validation_info.logon_time)}".indent(2)
+      output << "Logoff Time: #{present_time(validation_info.logoff_time)}".indent(2)
+      output << "Kick Off Time: #{present_time(validation_info.kick_off_time)}".indent(2)
+      output << "Password Last Set: #{present_time(validation_info.password_last_set)}".indent(2)
+      output << "Password Can Change: #{present_time(validation_info.password_can_change)}".indent(2)
+      output << "Password Must Change: #{present_time(validation_info.password_must_change)}".indent(2)
+
+      output << "Logon Count: #{validation_info.logon_count}".indent(2)
+      output << "Bad Password Count: #{validation_info.bad_password_count}".indent(2)
+      output << "User ID: #{validation_info.user_id}".indent(2)
+      output << "Primary Group ID: #{validation_info.primary_group_id}".indent(2)
+      output << "User Flags: #{validation_info.user_flags}".indent(2)
+      output << "User Session Key: #{present_user_session_key(validation_info.user_session_key)}".indent(2)
+      output << "User Account Control: #{validation_info.user_account_control}".indent(2)
+      output << "Sub Auth Status: #{validation_info.sub_auth_status}".indent(2)
+
+      output << "Last Successful Interactive Logon: #{present_time(validation_info.last_successful_i_logon)}".indent(2)
+      output << "Last Failed Interactive Logon: #{present_time(validation_info.last_failed_i_logon)}".indent(2)
+      output << "Failed Interactive Logon Count: #{validation_info.failed_i_logon_count}".indent(2)
+
+      output << "SID Count: #{validation_info.sid_count}".indent(2)
+      output << "Resource Group Count: #{validation_info.resource_group_count}".indent(2)
+
+      output << "Group Count: #{validation_info.group_count}".indent(2)
+      output << 'Group IDs:'.indent(2)
+      output << validation_info.group_memberships.map { |group| "Relative ID: #{group.relative_id}, Attributes: #{group.attributes}".indent(4) }
+
+      output << "Logon Domain ID: #{validation_info.logon_domain_id}".indent(2)
+
+      output << "Effective Name: #{present_rpc_unicode_string(validation_info.effective_name)}".indent(2) # unless effective_name_info.to_s.blank?
```

Comment:
```suggestion
      output << "Effective Name: #{present_rpc_unicode_string(validation_info.effective_name)}".indent(2)
```

And for the others below